src/app/models/quiz_attempt.py

```diff
@@ -45,8 +46,13 @@ def graded_categories(self):
 
     @classmethod
     def get_by_user_id(cls, user_id):
-        """ Get all quiz attempts from a user id """
-        return cls.query(cls.user_id == user_id).fetch()
+        """
+        Get all quiz attempts from a user id
+        :rtype: QuizAttempt or None
+        """
+        attempts = cls.query(cls.user_id == user_id).fetch()
+
+        return attempts[0] if attempts else None
```

Comment:
@gholtslander I don't remember if we did these kind of ternary expressions at LLC, but this will return the first item in the attempts variable that gets returned above, but only if that variable isn't `None` (Python's null value).